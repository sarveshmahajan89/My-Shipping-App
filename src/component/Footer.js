import React from "react";

class Footer  extends React.Component {
    render() {
        return (
            <footer className="text-center">
                <div className="footer-above">
                    <div className="container">
                        <div className="row">
                            <div className="footer-col col-md-12">
                                <h3>About My Shipment App</h3>
                                <p>My Shipment App is a professional services firm trusted by hundreds of leading brands for product shipment across all
                                    Globe.</p>
                            </div>
                        </div>
                    </div>
                </div>
                <div className="footer-below">
                    <div className="container">
                        <div className="row">
                            <div className="col-lg-12">
                                Copyright &copy; My Shipment App 2020
                            </div>
                        </div>
                    </div>
                </div>
            </footer>
        );
    }
}
export default Footer ;