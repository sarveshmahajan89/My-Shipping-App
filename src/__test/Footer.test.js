import React from 'react';

import Footer from '../component/Footer.js'

it('Footer renders correctly', () => {

    expect(<Footer />).toMatchSnapshot();
});
