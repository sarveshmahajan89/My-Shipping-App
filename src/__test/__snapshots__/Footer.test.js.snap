// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact renders correctly 1`] = `<Contact />`;

exports[`Footer renders correctly 1`] = `<Footer />`;
